<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RolliVROverlayWidget</class>
 <widget class="QWidget" name="RolliVROverlayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hello World Overlay</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#RolliVROverlayWidget {
	background: qlineargradient(x1:0 y1:0, x2:0 y2:1, stop:0 #31353d, stop:1 #292c33);	
	color: #dee2e5;
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}

QGroupBox::indicator {
	width: 22px;
	height: 22px;
}

QWidget {
	font: 10pt;
	color: #dee2e5;
}

QGroupBox {
	border: none;
}

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top;
}

QPushButton {
	border: none;
	padding: 8px;
	background-color: #3d4450;
}

QPushButton:hover {
	background-color: #515863;
}

QPushButton:pressed {
	background-color: #2f3540;
}

QLabel {
	color: #dee2e5;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="mainVerticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="startInfoLabel">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Forced movement should only be used if a VR application does not support custom controller bindings. Before pressing 'START FORCED MOVEMENT' take place on the platform and face forward to guarantee correct calibration.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startStopPushButton">
       <property name="text">
        <string>START FORCED MOVEMENT</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="hiddenWhileNotRunningWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="notRunningHiddenVerticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="resetPushButton">
          <property name="text">
           <string>RESET ACCUMULATED MOVEMENT</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="advancedConfigurationGroupBox">
          <property name="title">
           <string>ADVANCED SETTINGS</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>3</number>
           </property>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <property name="horizontalSpacing">
              <number>100</number>
             </property>
             <property name="verticalSpacing">
              <number>10</number>
             </property>
             <item row="0" column="1">
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>22</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="formHeightOffsetLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>HEIGHT ADJUSTMENT</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QPushButton" name="decrementHeightOffsetPushButton">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>&lt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="heightOffsetLabel">
                 <property name="text">
                  <string>0.00m</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="incrementHeightOffsetPushButton">
                 <property name="text">
                  <string>&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="fromSpeedLabeforml">
               <property name="text">
                <string>TURN SPEED</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="decrementTurnSpeedPushButton">
                 <property name="text">
                  <string>&lt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="turnSpeedLabel">
                 <property name="text">
                  <string>100%</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="incrementTurnSpeedPushButton">
                 <property name="text">
                  <string>&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="formmovementSpeedLabel">
               <property name="text">
                <string>MOVEMENT SPEED</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QPushButton" name="decrementMovementSpeedPushButton">
                 <property name="text">
                  <string>&lt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="movementSpeedLabel">
                 <property name="text">
                  <string>100%</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="incrementMovementSpeedPushButton">
                 <property name="text">
                  <string>&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
